### PR TITLE
update ci to run test on all platforms

### PR DESCRIPTION
**Pull request by valpinkman**

### CODE REVIEW

**Comment by gre** [comment]
windows are you drunk

**Comment by gre** [comment]
i don't understand why it fails on ubuntu neither

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "prettier": "prettier --write 'src/**/*.js'",
     "lint": "eslint src",
     "flow": "flow src",
-    "jest": "rm -rf libcoredb && mkdir libcoredb && cross-env TZ=America/New_York jest",
+    "jest": "npx rimraf libcoredb && mkdir libcoredb && cross-env TZ=America/New_York jest",
```
Comment (gre) [inline package.json]: better be a dev deps to not being installed each time we run.

Comment (valpinkman) [reply package.json]: ok

**Pull request merged**